terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -176,6 +200,7 @@
         "brennan@ucsc.edu"
     )
 ]
+# noqa lines: end
```

Comment:
There should be a conflict here, Noah added additional pubkeys in this section...

Reply:
The new keys didn't cause a conflict :shrug:

Reply:
w00t w00t.